app/assets/v2/js/shared.js

```diff
@@ -1356,6 +1356,7 @@ function newTokenTag(amount, tokenName, tooltipInfo, isCrowdfunded) {
   span.innerHTML = amount + ' ' + tokenName +
     (isCrowdfunded ? '<i class="fas fa-users ml-1"></i>' : '');
 
+  p.className = 'inner-tooltip';
```

Comment:
should we add this to the template? instead of JS

Reply:
yeah I definitely think so

Reply:
since this is for fixing bug, i prefer to make minimal changes. To use template instead of js demand refactoring code with more careful test